lib/mutant/mutator/node/regexp/character_type.rb

```diff
@@ -0,0 +1,30 @@
+module Mutant
+  class Mutator
+    class Node
+      module Regexp
+        # Character type mutator
+        class CharacterType < Node
+          extend Mutant::AST::Sexp
```

Comment:
This makes the AST DSL a public interface of this specific class. I wounder if we can do better.